src/main/java/svc/data/citations/datasources/tyler/TylerCitationDataSource.java

```diff
@@ -1,59 +1,106 @@
 package svc.data.citations.datasources.tyler;
 
+import java.net.URI;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.ParameterizedTypeReference;
 import org.springframework.http.HttpEntity;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpMethod;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Repository;
+import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.RestTemplate;
 import org.springframework.web.util.UriComponentsBuilder;
+
 import svc.data.citations.CitationDataSource;
 import svc.models.Citation;
 
-import java.sql.Date;
-import java.time.LocalDate;
-import java.util.List;
-
 @Repository
 public class TylerCitationDataSource implements CitationDataSource {
 
 	@Autowired
 	private TylerConfiguration tylerConfiguration;
 
 	@Autowired
+	private TylerTransformer tylerTransformer;
+
+	@Autowired
 	private RestTemplate restTemplate;
 
-	@SuppressWarnings("unused")
+	private DateTimeFormatter dobFormatter = DateTimeFormatter.ofPattern("MM/dd/uuuu");
+
 	@Override
 	public List<Citation> getByCitationNumberAndDOB(String citationNumber, LocalDate dob) {
 
 		UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(tylerConfiguration.rootUrl)
-				.queryParam("citationNumber", citationNumber).queryParam("dob", Date.valueOf(dob));
+				.queryParam("citationNumber", citationNumber).queryParam("dob", dob.format(dobFormatter));
 		HttpHeaders headers = new HttpHeaders();
 		headers.add("apikey", tylerConfiguration.apiKey);
 		HttpEntity<?> query = new HttpEntity<>(headers);
 		ResponseEntity<List<TylerCitation>> tylerCitationsResponse = null;
 		ParameterizedTypeReference<List<TylerCitation>> type = new ParameterizedTypeReference<List<TylerCitation>>() {
 		};
-		tylerCitationsResponse = restTemplate.exchange(builder.build().encode().toUri(), HttpMethod.GET, query, type);
-		List<TylerCitation> tylerCitations = tylerCitationsResponse.getBody();
-
-		// TODO
-		// This is close. It's getting 400s. Probably malformed requests.
-		// I give up for the night. Your turn.
+		URI uri = builder.build().encode().toUri();
+		List<TylerCitation> tylerCitations = null;
+		try {
+			tylerCitationsResponse = restTemplate.exchange(uri, HttpMethod.GET, query, type);
+			tylerCitations = tylerCitationsResponse.getBody();
+		} catch (HttpClientErrorException ex) {
+			System.out.println(ex.getMessage());
+			System.out.print(ex.getResponseBodyAsString());
+		}
 
-		return null;
+		return tylerTransformer.citationsFromTylerCitations(tylerCitations);
```

Comment:
This thing is already called the TylerTransformer so the method problem doesn't need the word Tyler in it. tylerTransformer.convertCitations() is probably sufficient or something.